source/utils/types.ts

```diff
@@ -77,6 +93,10 @@ export interface Options extends RequestOptions {
 	method?: Method;
 	retry?: number | RetryOption;
 	throwHttpErrors?: boolean;
+	cookieJar?: CookieJar;
+	request?: RequestFn;
+	agent: https.Agent | boolean | { [key: string]: https.Agent };
+	gotTimeout?: number | Delays;
```

Comment:
I tried adding `cache?: Store<any>` here, though the linter then required pulling in `keyv` as a dependency. I think a better solution would be to export Store as a typealias from the typings for CacheableRequest.

There's another issue with the typings for CacheableRequest, related to whether or not it's a default export. I added a todo comment. I suspect a change is needed to the typings, though I'm afraid it's a bit over my head.

Reply:
> I think a better solution would be to export Store as a typealias from the typings for CacheableRequest.

👍

Reply:
I opened DefinitelyTyped/DefinitelyTyped#34155 though it's not blocking this.

Reply:
Merged and updated.